MREUnityRuntime/MREUnityRuntimeLib/Behaviors/Actions/BehaviorActionHandler.cs

```diff
@@ -29,12 +28,12 @@ internal sealed class BehaviorActionHandler : IActionHandler
 			_attachedActorId = attachedActorId;
 		}
 
-		void IActionHandler.HandleActionPerforming(IUser user, BaseActionData actionData)
+		internal void HandleActionPerforming(IUser user, BaseActionData actionData)
```

Comment:
Do we need this method? Seems like a pretty trivial wrapper.

Reply:
It was necessary when we had the IActionHandler interface before.  Not needed now.

---
In reply to: [434215490](https://github.com/microsoft/mixed-reality-extension-unity/pull/160#discussion_r434215490) [](ancestors = 434215490)

Reply:
It was added more for the simplicity of the caller to not have to pass in "Performing" for the old and new state in order for the event to be handled correctly.  Is more of an elegance for the API then it just being a simple wrapper.

---
In reply to: [434215490](https://github.com/microsoft/mixed-reality-extension-unity/pull/160#discussion_r434215490) [](ancestors = 434215490)